AnkiDroid/src/main/java/com/ichi2/utils/AlertDialogFacade.kt

```diff
@@ -202,30 +203,63 @@ fun AlertDialog.Builder.customListAdapter(adapter: RecyclerView.Adapter<*>) {
 }
 
 /**
+ * @param hint The hint text to be displayed to the user
  * @param prefill The text to initially appear in the [EditText]
  * @param allowEmpty If true, [DialogInterface.BUTTON_POSITIVE] is disabled if the [EditText] is empty
  * @param displayKeyboard Whether to open the keyboard when the dialog appears
- * @param callback called whenever the text is changed
+ * @param callback if [waitForPositiveButton], called when [positiveButton] is pressed, otherwise
+ * @param maxLength if set, the user may not enter more than the supplied number of digits
+ * @param inputType see [EditText.setInputType]
+ * called whenever the text is changed
+ * @param waitForPositiveButton MaterialDialog compat: if `false` [callback] is called on input
+ * if `true` [callback] is called when [positiveButton] is pressed
```

Comment:
method param docs mis-interleaved just a bit

```suggestion
 * @param callback if [waitForPositiveButton], called when [positiveButton] is pressed, otherwise
 * called whenever the text is changed
 * @param maxLength if set, the user may not enter more than the supplied number of digits
 * @param inputType see [EditText.setInputType]
 * @param waitForPositiveButton MaterialDialog compat: if `false` [callback] is called on input
 * if `true` [callback] is called when [positiveButton] is pressed
```

Reply:
> `@param callback if [waitForPositiveButton], called when [positiveButton] is pressed, otherwise` is now a fragment